### ISSUE TITLE
Add Time Zone configuration to Dockerfile

### ISSUE THREAD

**Issue by frostyandy2k** [ScaleIT-Org/nodejs-app-skeleton#6]
```
Current default time zone: 'Etc/UTC'
Local time is now:      Wed Feb 14 13:07:36 UTC 2018.
Universal Time is now:  Wed Feb 14 13:07:36 UTC 2018.
Run 'dpkg-reconfigure tzdata' if you wish to change it.
```

**Comment by Kuhno92** [comment]
Fixed with 4dff6e26c88b7bed15b166ed3237d5c9478b0bde

**Comment by frostyandy2k** [comment]
Don't forget to commit changes to the alpine branch. You can use the following in the future:
```
git checkout alpine
git pull --rebase origin master
```
Done for the last patches.
